openmc/filter.py

```diff
@@ -576,6 +655,28 @@ class SurfaceFilter(IntegralFilter):
         filter's bins.
 
     """
+    @property
+    def bins(self):
+        return self._bins
+
+    @property
+    def num_bins(self):
+        return len(self.bins)
+
+    @bins.setter
+    def bins(self, bins):
+        # Format the bins as a 1D numpy array.
+        bins = np.atleast_1d(bins)
+
+        # Check the bin values.
```

Comment:
You can't actually give a surface filter to OpenMC.  The only way to get a surface filter is to ask for a current tally on a mesh and one will appear in the statepoint.